conf.yaml

```diff
@@ -4,6 +4,12 @@ paths:
     branch_tracker: html/branches.yaml
 
 repos:
+    stack:
+        url:        https://github.com/elastic/stack-docs.git
+        current:    master
```

Comment:
(We talked on Slack: I think we should wait to start versioning once we're sure that it is stable, then fix it).